index.php

```diff
@@ -310,11 +347,11 @@ function generateCAPassword($password) {
                 </md-input-container>
               </form>
             </md-tab>
-            <md-tab label="LAN Password">
+            <md-tab label="802.1X Password">
               <form name="LANPasswordResetForm" flex style="padding: 20px" action="./?lan-reset" method="POST">
                 <div layout="row">
                   <md-input-container flex>
-                    <label>Email</label>
+                    <label>Username</label>
```

Comment:
People confuse username for email and fill that. 

People also confuse email for full email and ignore the drop down beside it. But that's another thing.